src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js

```diff
@@ -184,6 +184,14 @@ export default React.createClass({
         } else if (this.state.backupInfo === null) {
             title = _t("Error");
             content = _t("No backup found!");
+        } else if (this.state.recoverInfo && this.state.recoverInfo.imported === 0) {
+            title = _t("Error Restoring Backup");
+            let failedToDecrypt;
+            content = <div>
+                <p>{_t(
+                    "Backup restore failed: please verify that you entered the correct recovery key.",
```

Comment:
Does the "Backup restore failed: " prefix seem redundant given the dialog title "Error Restoring Backup"?

Reply:
oh yes, it is a bit - adjusted to just say it can't decrypt the backup